package/yast2-registration.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Nov  6 13:56:48 UTC 2018 - dgonzalez@suse.com
+
+- Avoid an useless error message when the SCC returns a 422 error
+  but the module is in a "silent mode" (bsc#1091825).
```

Comment:
That "silent mode" is probably a bit confusing for the end users. I'd use more user oriented text (the technical details can be found in Git sources and history).

Something like 
```
Do not display a connection error when automatically trying to use the base product
registration code for the extensions.
```